tools/nni_cmd/config_schema.py

```diff
@@ -82,6 +82,13 @@
         Optional('start_step'): And(int, lambda x: 0 <= x <= 9999)
     },
     Optional('gpuNum'): And(int, lambda x: 0 <= x <= 99999)
+},{
+    'builtinAssessorName': lambda x: x in ['Curvefitting'],
+    Optional('classArgs'): {
+        Optional('start_step'): And(int, lambda x: 0 <= x <= 9999),
+        'epoch_num': And(int, lambda x: 0 <= x <= 9999)
```

Comment:
would be better to put required before optional.

Reply:
Done.